src/blob.rs

```diff
@@ -418,6 +425,22 @@ impl<'a> BlobObject<'a> {
 
         Ok(())
     }
+
+    pub fn get_exif_orientation(&self) -> Result<i32, Error> {
+        let file = std::fs::File::open(self.to_abs_path())?;
+        let mut bufreader = std::io::BufReader::new(&file);
+        let exifreader = exif::Reader::new();
+        let exif = exifreader.read_from_container(&mut bufreader)?;
+        if let Some(orientation) = exif.get_field(exif::Tag::Orientation, exif::In::PRIMARY) {
+            match orientation.value.get_uint(0) {
+                Some(3) => return Ok(180),
```

Comment:
Had to search for these constants and found http://sylvana.net/jpegcrop/exif_orientation.html to understand it

Maybe explain in a comment that flipping is not supported, and `warn!` about unsupported values.

Reply:
sorry for the missing link. in fact, the [original documentation](https://www.digicamsoft.com/exif22/exif22/html/exif22_1.htm) is so bad that i was happy to put it aside :)

i will add your link to a comment and also add warning.